src/storage.js

```diff
@@ -138,12 +147,25 @@ class AWSStorage {
     const body = await res.buffer();
     const key = `${prefix}${path}`;
 
-    const result = await this.client.send(new PutObjectCommand({
+    const input = {
       Body: body,
       Bucket: this.bucket,
-      ContentType: res.headers.get('content-type'),
+      Metadata: {},
       Key: key,
-    }));
+    };
+
+    Array.from(res.headers.entries()).forEach(([name, value]) => {
```

Comment:
isn't entries a `Map()` entries and already has a forEach ?
```suggestion
    res.headers.entries().forEach(([name, value]) => {
```

Reply:
No, res.headers.entries() is an iterator